st2client/st2client/commands/action.py

```diff
@@ -975,6 +980,11 @@ def run(self, args, **kwargs):
             kwargs['timestamp_gt'] = args.timestamp_gt
         if args.timestamp_lt:
             kwargs['timestamp_lt'] = args.timestamp_lt
+        if args.sort_order:
+            if args.sort_order in ['asc', 'ascending']:
```

Comment:
I have a feeling this asc and desc might get confusing for user. So maybe add help strings - "earliest first" and "latest first" will make it less so.